### DISCUSSION TITLE
v0.0.9 "with update baseFolderNode"

### DISCUSSION THREAD

**Discussion by EdoFro** [EdoFro/Freeplane_Map-Drive-Inator#25] [Announcements]
## v0.0.9
* Now it recognizes when link of baseFolder node points to an unexisting file/folder
* some refactoring
* central statusInfo method for MDI
* "create base folder node" can now be used to update baseFolderNode parameters
* changed baseFolderNode note to markdown format

<hr /><em>This discussion was created from the release <a href='https://github.com/EdoFro/Freeplane_Map-Drive-Inator/releases/tag/v0.0.9'>v0.0.9 "with update baseFolderNode"</a>.</em>

**Comment by EdoFro** [comment]
the important changes are:
- Now it recognizes when link of baseFolder node points to an unexisting file/folder
  - this helps, when changing the location of a map, to see if the link in the baseFolder node really points to an existing folder
- "create base folder node" can now be used to **update** baseFolderNode parameters
  - this way it's easier to change the parameters from the baseFolder node

Please report back any issue, idea, opinion.

BR
edo

**Comment by EdoFro** [comment]
I had a wrong version of the addon in my computer (I messed up with github and changing branch).
I changed now the .addon.mm file to be sure that it's the correct one.

**Reply by bepolymathe** [reply]
Ok, thanks ! I’ll test it tomarrow morning with both machines.

**Reply by bepolymathe** [reply]
I just tried it quickly but I have something new. Under linux, no worries. Under windows, the same folder has the right link (if I click on the folder I go to the right place) but the MDI command doesn't seem to work but doesn't return an error message either. 🤔

**Reply by EdoFro** [reply]
hi @bepolymathe,
can you please submit an issue and explain a little how to reproduce the problem?
thanks

**Reply by bepolymathe** [reply]
Yes of course but I'm a bit far from my windows computer for two days. I'll do this as soon as I can.
